Website/src/rendering/src/stories/Checkout/GiftCheckboxLineItem.stories.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import GiftCheckboxLineItem from '../../components/Checkout/GiftCheckboxLineItem';
 import { MockStore } from '../mock-store';
+import { cartSlice } from './CheckoutCommon';
 
 export default {
   title: 'Components/Checkout/GiftCheckbox',
```

Comment:
Could the title of this story please match the component name: `GiftCheckboxLineItem` ?